rules/library.bzl

```diff
@@ -506,7 +506,10 @@ def apple_library(name, library_tools = {}, export_private_headers = True, names
             generated_header_name = generated_swift_header_name,
             srcs = swift_sources,
             copts = swift_copts,
-            deps = deps + internal_deps + lib_names,
+            deps = deps + internal_deps + lib_names + select({
```

Comment:
do we really want to do this for every `swift_library` ? Ideally, it'd only be done for `ios_application` and `ios_unit_test`, so _most_ work can be shared between the flag being in either state

Reply:
Let me see if there's a way to do that. thanks!

Reply:
I updated this macro to accept a parameter for whether or not to add swift local debug options.